### ISSUE TITLE
Neovide attempting to open GUI when trying to run in wsl mode or help 

### ISSUE THREAD

**Issue by Lite5h4dow** [Kethku/neovide#562]
I am attempting to run neovide in wsl mode and get the help dialog in the console so i can see how to use it properly, however i get the following error as it is trying to render the GUI for some reason.

 ```
$ neovide --wsl
Ignored client type property: "methods"
Ignored client type property: "attributes"

error: XDG_RUNTIME_DIR not set in the environment.
The path /dev/dri/ cannot be opened or is not available
The path /dev/dri/ cannot be opened or is not available
thread 'main' panicked at 'Failed to create sdl video subsystem: "No available video device"', src/window/sdl2/mod.rs:432:10
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by Kethku** [comment]
This seems like a graphics driver issue. What gpu/integrated graphics do you have? Can you try the opengl branch?

**Comment by Lite5h4dow** [comment]
im running a NVidia GTX 1070 gpu. ill give the openGL branch a go

**Comment by Kethku** [comment]
Ah that explains the issue. Neovide is just the front end for neovim. To connect to the remote process, you have to run neovim like this: `nvim --headless --listen localhost:6666` or just run neovide from the windows side using the --wsl command line argument which will start a neovim process in wsl land.

TLDR: install neovide on the windows side and neovim on the wsl side and the --wsl arg should work.

**Comment by Lite5h4dow** [comment]
current steps. running `neovide --wsl --log` worked as expected. this is the result from the log:
```
INFO [neovide::bridge] Close watcher started
ERROR [neovide::bridge] Neovide requires version 0.4 or higher
```
however if i run `nvim -v` on windows i get the following:
```
NVIM v0.5.0-dev+1256-gd19348923
Build type: RelWithDebInfo
LuaJIT 2.0.5
Compilation: /usr/sbin/cc -D_FORTIFY_SOURCE=2 -march=x86-64 -mtune=generic -O2 -pipe -fno-plt -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1 -O2 -g -Og -g -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wshadow -Wconversion -Wmissing-prototypes -Wimplicit-fallthrough -Wvla -fstack-protector-strong -fno-common -fdiagnostics-color=always -DINCLUDE_GENERATED_DECLARATIONS -D_GNU_SOURCE -DNVIM_MSGPACK_HAS_FLOAT32 -DNVIM_UNIBI_HAS_VAR_FROM -DMIN_LOG_LEVEL=3 -I/home/lotus/.cache/yay/neovim-nightly-git/src/build/config -I/home/lotus/.cache/yay/neovim-nightly-git/src/neovim-nightly-git/src -I/usr/include -I/home/lotus/.cache/yay/neovim-nightly-git/src/build/src/nvim/auto -I/home/lotus/.cache/yay/neovim-nightly-git/src/build/include
Compiled by lotus@DESKTOP-89770M2

Features: +acl +iconv +tui
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/usr/share/nvim"

Run :checkhealth for more info
```

**Comment by Kethku** [comment]
Recapping for posterity: no idea why the release build or built versions from actions didn't work, but a manual build on the users machine fixed the problem and caused wsl mode to work normally.
